sysutils/rawhide/Portfile

```
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           makefile 1.0
+PortGroup           legacysupport 1.0
+
+name                rawhide
+version             3.2
+
+description         (rh) find files using pretty C expressions
+long_description    (rh) find files using pretty C expressions
```

Use `{*}${description}` instead of copying:
```suggestion
long_description    {*}${description}
```

---

Thanks. I've done that, and also made the description actually longer.